explainer.md

```diff
@@ -600,6 +605,18 @@ enum XREnvironmentBlendMode {
   "alpha-blend",
 };
 
+dictionary XRRenderStateOptions {
+  double depthNear;
+  double depthFar;
+  XRLayer baseLayer;
+};
+
+[SecureContext, Exposed=Window] interface XRRenderState {
+  readonly attribute double depthNear;
+  readonly attribute double depthFar;
+  readonly attribute XRLayer baseLayer;
```

Comment:
Per the comment on the PR that was just filed by @Manishearth, this should be nullable, yes?